tests/unit/plugins/modules/test_maskingview.py

```diff
@@ -9,7 +9,7 @@
 __metaclass__ = type
 
 import pytest
-from mock.mock import MagicMock
+from mock.mock import MagicMock, patch
```

Comment:
>The statement from mock.mock import MagicMock is considered obsolete because the mock library has been integrated into the Python standard library as unittest.mock since Python 3.3. 

Do we plan to replace `mock.mock` with `unittest.mock`?

Reply:
hi @baoy1 , good point...
for now, the powermax ut are all still using  mock.mock . maybe we can raise one standalone ticket to  replace mock.mock with unittest.mock

@P-Cao @RayLiu7 FYI

Reply:
@taohe1012 @baoy1 Agree. I believe we need to address this in future. For now, we can note it down.